packages/phrases/src/locales/en.ts

```diff
@@ -167,7 +167,7 @@ const translation = {
       token_endpoint: 'Token Endpoint',
       user_info_endpoint: 'User Info Endpoint',
       save_changes: 'Save Changes',
-      more_options: 'More Options',
+      more_options: 'MORE OPTIONS',
```

Comment:
should we change to uppercase in code instead of here?

Reply:
good call, or maybe use a non-case-sensitive font, created an issue LOG-2311 and will discuss offline with the designer.